gridstatus/ercot.py

```diff
@@ -2106,7 +2105,10 @@ def _handle_unplanned_resource_outages_file(self, xls):
             skipfooter=1,
         )
 
-        df.insert(0, "Report Time", as_of)
+        # Already has local timezone
+        df["Publish Time"] = pd.to_datetime(doc.publish_date)
+
+        df.insert(0, "Current As Of", as_of)
```

Comment:
The reasoning behind renaming this from `Report Time` to `Current As Of` is ERCOT calls this "Current as of" in the data. `Report Time` seems ambiguous compared to `Current As Of`.